cfg/samples.cfg

```diff
@@ -1,4 +1,4 @@
-path : /home/xuan/coffea/trees/
+#path : /home/xuan/coffea/trees/
```

Comment:
@GonzalezFJR what would you recommend to do here? Can we comment it out or better to remove it?